public/index.js

```diff
@@ -1,23 +1,31 @@
 const user = {};
 
-const addChatMessageToDOM = (nickname, message) => {
+const addMessageToDOM = message => {
   const listItem = document.createElement('li');
-  listItem.textContent = `${nickname}: ${message}`;
+  if (message.type == 'chat') {
+    listItem.textContent = `${message.nickname}: ${message.body}`;
+  } else if (message.type == 'system') {
+    listItem.textContent = message.body;
+    listItem.style.fontStyle = 'italic';
+  }
   document.getElementById('messages').appendChild(listItem);
 };
 
 document.addEventListener('DOMContentLoaded', () => {
   const socket = io();
+  let message = {};
 
-  socket.on('system message', message => {
-    const listItem = document.createElement('li');
-    listItem.style.fontStyle = 'italic';
-    listItem.textContent = message;
-    document.getElementById('messages').appendChild(listItem);
+  socket.on('system message', messageBody => {
```

Comment:
You should set a standard on the message where all messages are objects, and every time you emit/listen to message events (chat and system) pass a message object.

Reply:
And `messageBody` could be called just `body`, because it will live within the `message` object.